base/applications/regedit/find.c

```diff
@@ -88,14 +88,25 @@ static BOOL CompareName(LPCWSTR pszName1, LPCWSTR pszName2)
 }
 
 static BOOL
-CompareData(
-    DWORD   dwType,
-    LPCWSTR psz1,
-    LPCWSTR psz2)
+CompareData(LPCWSTR pszSubKey, LPCWSTR pszValueName, DWORD dwType, LPCVOID pv1, size_t cb1)
 {
-    INT i, cch1 = wcslen(psz1), cch2 = wcslen(psz2);
-    if (dwType == REG_SZ || dwType == REG_EXPAND_SZ)
+    if (dwType == REG_SZ || dwType == REG_EXPAND_SZ || dwType == REG_MULTI_SZ)
     {
+        LPCWSTR psz1 = pv1; /* We assume pv1 is UNICODE_NULL-terminated */
```

Comment:
If you can assume that, why are you passing `cb1` to the function?

Reply:
Assumption is removed.